graylog2-web-interface/src/components/lookup-tables/LookupTableForm.tsx

```diff
@@ -217,10 +221,20 @@ const LookupTableForm = ({ saved, create, table }: Props) => {
           <fieldset>
             <Row>
               <Col mdOffset={3} md={9}>
-                {create ? (
-                  <Button type="submit" bsStyle="success">Create Lookup Table</Button>
-                ) : (!loadingScopePermissions && scopePermissions?.is_mutable) && (
-                  <Button type="submit" bsStyle="success">Update Lookup Table</Button>
+                {create && (
+                  <FormSubmit submitButtonText="Create lookup table"
+                              submitLoadingText="Creating lookup table..."
+                              isSubmitting={isSubmitting}
+                              isAsyncSubmit
+                              onCancel={onCancel} />
+                )}
+
+                {(!create && !loadingScopePermissions && scopePermissions?.is_mutable) && (
```

Comment:
we can create some var smth like `updatable = !create && !loadingScopePermissions && scopePermissions?.is_mutable` will be a bit cleaner